src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -266,29 +262,31 @@ protected void onCreate(Bundle savedInstanceState)
                 Logger.v(TAG, "Caller needs to be verified using special redirectUri");
                 mRedirectUrl = PackageHelper.getBrokerRedirectUrl(mCallingPackage, signatureDigest);
             }
-            Logger.v(TAG, "OnCreate redirectUrl:" + mRedirectUrl + " startUrl:" + mStartUrl
-                    + " calling package:" + mCallingPackage + " signatureDigest:" + signatureDigest
-                    + " current Context Package: " + getPackageName());
-        }
-        else
-        {
-            Logger.v(TAG + methodName, "Non-broker request for package " + getCallingPackage());
+
+            Logger.v(TAG, "Broker redirectUrl: " + mRedirectUrl + " The calling package is: " + mCallingPackage 
```

Comment:
it could, we put it in the additional part. If it's ok for app to log upn, they'll read the additional message.